### PR TITLE
Update subcscription

### PR DESCRIPTION
**Pull request by hammadjaved686**

### CODE REVIEW

File: src/app/product/add-product/add-product.component.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<form [formGroup]="productForm" (ngSubmit)="submitProduct()">
+    <!-- Form fields for product details -->
+    <input formControlName="title" placeholder="Title">
```
Comment (zubin00) [inline src/app/product/add-product/add-product.component.html:3]: Please add a proper validation message here

File: src/app/product/add-product/add-product.component.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      images: [[]]
+    });
+  }
+
+  submitProduct(): void {
+    if (this.productForm.valid) {
+      const newProduct = this.productForm.value;
+      console.log('product : ', newProduct)
+      newProduct.images = ["https://placeimg.com/640/480/any"]
+      this.http.post('https://api.escuelajs.co/api/v1/products/', newProduct).subscribe((response) => {
```
Comment (zubin00) [inline src/app/product/add-product/add-product.component.ts]: do not use any hard-coded url inside ts component please make a separate constant file for it

Comment (hammadjaved686) [reply src/app/product/add-product/add-product.component.ts]: noted

File: src/app/product/add-product/add-product.component.html (hunk 1 above)
Comment (hammadjaved686) [reply src/app/product/add-product/add-product.component.html:3]: okay

**Pull request merged**